src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html

```diff
@@ -142,7 +141,8 @@
             </div>
             <div class="right-editor">
                 <pre id="docEditorRight" class="form-control absolute-fill"
-                     data-bind="aceEditor: { code: documentTextRight, fontSize:'16px', lang: 'ace/mode/raven_document', readOnly: inReadOnlyMode() || inDiffMode() }, disable: isBusy, validationOptions: { errorsAsTitle: false }, validationElement: documentText, visible: !isDeleteRevision()"></pre>
+                     data-bind="aceEditor: { code: documentTextRight, fontSize:'16px', lang: 'ace/mode/raven_document', readOnly: inReadOnlyMode() || inDiffMode() }, 
```

Comment:
does it work properly? I remember knockout has issues with bindings are spread across multiple lines.

Reply:
From all tests that I have done, all seems to work properly.